src/javascript/app/pages/trade/barriers.js

```diff
@@ -160,14 +161,27 @@ const Barriers = (() => {
         Defaults.remove('barrier', 'barrier_high', 'barrier_low');
     };
 
+    /**
+    * Validate Barriers
+    */
     const validateBarrier = () => {
-        const barrier_element = getElementById('barrier');
-        const empty           = isNaN(parseFloat(barrier_element.value)) || parseFloat(barrier_element.value) === 0;
+        const barrier_element      = getElementById('barrier');
+        const empty                = isNaN(parseFloat(barrier_element.value))||parseFloat(barrier_element.value) === 0;
+        const barrier_high_element = getElementById('barrier_high');
+        const barrier_low_element  = getElementById('barrier_low');
```

Comment:
do we need `barrier_low_element`?

Reply:
Yes. Shouldn't we compare the values of low barrier and high barrier?

Reply:
it can move inside the if condition as well